protobuf-codegen/src/gen/mod_rs.rs

```diff
@@ -3,9 +3,14 @@ use crate::gen::code_writer::CodeWriter;
 
 pub(crate) fn gen_mod_rs(mods: &[String]) -> compiler_plugin::GenResult {
     let v = CodeWriter::with_no_error(|w| {
-        w.comment(&format!("{}generated", "@"));
-        w.write_line("");
+        w.comment(&format!(
+            "This file is generated by {} {}. Do not edit",
+            env!("CARGO_PKG_NAME"),
```

Comment:
Code generator is not necessarily executed by cargo. See how similar code is inserted in other files.

Reply:
Removed